packages/property-grid/src/components/PropertyGrid.tsx

```diff
@@ -28,6 +28,7 @@ import {
   PropertyGrid as CorePropertyGrid,
```

Comment:
Is this import even being used anymore? If not, you can remove it. PropertyGrid as CorePropertyGrid